orderer/common/follower/follower_chain.go

```diff
@@ -313,7 +329,7 @@ func (c *Chain) pull() error {
 			// Trigger creation of a new consensus.Chain.
 			c.logger.Info("On-boarding finished successfully, invoking chain creation callback")
 			c.halt()
-			c.chainCreationCallback()
+			c.chainCreator.SwitchFollowerToChain(c.ledgerResources.ChannelID())
```

Comment:
+1 I like how it's now clear what's happening instead of the generic "chainCreationCallback". :)